config/common.go

```diff
@@ -1,6 +1,7 @@
+//go:generate go run config_generator/main.go config.json
+
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See License.txt for license information.
-
```

Comment:
Was this newline removed accidentally?

Reply:
@reflog, not sure if you meant to resolve these conversations before addressing the comments?